i18n/en.js

```diff
@@ -56,17 +56,17 @@ export default {
     submit: "Submit",
   },
   skillLevels: {
-    1: "I don't know it",
-    2: "I know a bit",
-    3: "I alredy tested/used it a little",
-    4: "I used it a lot",
-    5: "I'm a pro",
+    1: "I looked at what it was",
+    2: "I played with it, I tested it",
+    3: "I have worked with it on significant projects",
```

Comment:
Cette option est la seule a ne pas utiliser le contraction. Je suggère d'aligner.
```suggestion
    3: "I've worked with it on significant projects",
```